apps/src/templates/FontAwesome.jsx

```diff
@@ -10,7 +10,7 @@ import _ from 'lodash';
  */
 export default function FontAwesome({icon, className, ...props}) {
   const newProps = _.assign({}, props, {
-    className: `fa fa-${icon} ${className}`
+    className: `fa fa-${icon} ${className ? className : ''}`
```

Comment:
It would probably be appropriate to refactor this whole line to use the `classnames` helper.